setup.py

```diff
@@ -63,6 +63,7 @@ def run(self, *arg, **kw):
     'boto3',
     'CherryPy',
     'click',
+    'dogpile.cache',
```

Comment:
Should this also be added to `requirements.txt`?  If so, then `requirements.txt` looks like it is missing `click`, `numpy`, and `ctk-cli` from older changes, too.